### DISCUSSION TITLE
멀티 플레이 애니메이션 동기화 문제 (헤어)

### DISCUSSION THREAD

**Discussion by smilejsu82** [naverz/zepeto-studio-kor#586] [5. Q&A]
동기화 하는데는 문제가 없는데요 
달려갈때 휘날리는 머리카락이 찌그러져요....ㅠ
![image](https://user-images.githubusercontent.com/50188795/161719235-d3d03a09-eb28-4031-b30a-9dbeb2f9a71a.png)

영상첨부 합니다 ~ 

https://user-images.githubusercontent.com/50188795/161720654-31e69342-c971-46d5-92b7-593959d775a0.mp4

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요. #147 과 동일한 현상으로 보여지며 확인 중에 있습니다.
